src/client/js/dialog.js

```diff
@@ -67,6 +70,7 @@ async function openDialog (path) {
     }
   } catch (e) {
     dialogEl.close()
+    window.gtag('event', 'Dialog', { action: 'open', result: 'failed', page_location: location.href })
```

Comment:
Do we want to add a ref to the actual dialog content being called or nah?  I don't mind either way, but just future-casting when a page_location has multiple dialogs and GA can't differentiate.  🤷 

e.g.: `window.gtag('event', 'Dialog', { action: 'open', result: 'failed', page_location: location.href, dialog_content: partialName })`

Reply:
Thanks!